### PR TITLE
Add Trigger Rally

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://sourceforge.net/p/trigger-rally/patches/15/
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://sourceforge.net/p/trigger-rally/patches/14/ https://sourceforge.net/p/trigger-rally/patches/15/

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build io.sourceforge.trigger_rally.TriggerRally

**Comment by Mailaender** [comment]
bot, build io.sourceforge.trigger_rally.TriggerRally

File: io.sourceforge.trigger_rally.TriggerRally.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+app-id: io.sourceforge.trigger_rally.TriggerRally
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: trigger-rally
+rename-icon: trigger-rally
+rename-desktop-file: trigger-rally.desktop
+rename-appdata-file: trigger-rally.appdata.xml
```
Comment (hfiguiere) [inline io.sourceforge.trigger_rally.TriggerRally.yaml:8]: there is no reason to have these rename if the files are in the flathub repository. I think I have mentionned this a few times already.

File: io.sourceforge.trigger_rally.TriggerRally.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    post-install:
+      - install -Dm755 ${FLATPAK_DEST}/share/games/trigger-rally/icon/trigger-64.png ${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/trigger-rally.png
+      - install -Dm755 ${FLATPAK_DEST}/share/games/trigger-rally/icon/trigger-128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/trigger-rally.png
+      - install -Dm755 ${FLATPAK_DEST}/share/games/trigger-rally/icon/trigger-256.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/trigger-rally.png
+      - install -Dm755 ${FLATPAK_DEST}/share/games/trigger-rally/icon/trigger-rally-icons.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/trigger-rally.svg
+      - install -Dm644 trigger-rally.appdata.xml ${FLATPAK_DEST}/share/appdata/trigger-rally.appdata.xml
+      - install -Dm644 trigger-rally.desktop ${FLATPAK_DEST}/share/applications/trigger-rally.desktop
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/project/trigger-rally/trigger-0.6.6.1/trigger-rally-0.6.6.1.tar.gz
```
Comment (hfiguiere) [inline io.sourceforge.trigger_rally.TriggerRally.yaml:55]:
```suggestion
        url: https://downloads.sourceforge.net/project/trigger-rally/trigger-0.6.6.1/trigger-rally-0.6.6.1.tar.gz
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: io.sourceforge.trigger_rally.TriggerRally.yaml (hunk 1 above)
Comment (Mailaender) [reply io.sourceforge.trigger_rally.TriggerRally.yaml:8]: They are already commited using this name. https://sourceforge.net/p/trigger-rally/code/HEAD/tree/data/icon/

**Comment by barthalion** [comment]
bot, build io.sourceforge.trigger_rally.TriggerRally

**Comment by Mailaender** [comment]
> • url-not-found         : <screenshot> failed to connect: SSL handshake failed [https://trigger-rally.sourceforge.io/img/trigger_screenshot_3.jpg]
• url-not-found         : <screenshot> failed to connect: SSL handshake failed [https://trigger-rally.sourceforge.io/img/trigger_screenshot_4.jpg]
• url-not-found         : <screenshot> failed to connect: SSL handshake failed [https://trigger-rally.sourceforge.io/img/trigger_screenshot_5.jpg]

This is clearly a @flathub bug. I also noticed in https://github.com/flathub/io.github.openhv.OpenHV that your build bots have frequent connection problems. It seems to get worse.

**Comment by barthalion** [comment]
bot, build io.sourceforge.trigger_rally.TriggerRally

**Pull request closed without merging**